lib/axon.ex

```diff
@@ -1528,10 +1528,34 @@ defmodule Axon do
   end
 
   @doc """
-  Splits input graph into a container of `n` input graphs
-  along the given axis.
+  Splits input graph into a container of `m` other input graphs 
```

Comment:
Does it work on all Axon graphs? If so, the usage of "input graph" here is kind of confusing.
If it's specific, I'd like to know more about what an input graph actually is and how it differs from other graphs

Reply:
I performed some tests but I cannot answer for all the possible Axon graph.
Do you have in mind some specific test case? In so please describe it to me so I can test it.
Thank you.

Reply:
@stefkohub I asked because I'm not _that_ familiar with Axon, but I noticed that the whole docstring refers to "input graphs". So either it is correct and I'll learn something, or we should change it to something like "Splits the input graph into a container of `n` other graphs", and change accordingly below as well

Reply:
In this case it is okay, `input graph` refers to any Axon object :)

Reply:
Thank you @seanmor5 and @polvalente I am the first who needs to learn here :)